### DISCUSSION TITLE
NoScribe 0.4.3 x86_64 doesn't open in MacOS Monterey 12.7.2 after security disabled

### DISCUSSION THREAD

**Discussion by buhro** [kaixxx/noScribe#42] [General]
Hello! After downloading noScribe 0.4.3 for Mac intel and following every step of installation, and security checks, it doesn't open. There ist no message of mistake or anything else. The picture below shows the last message of mistake before nothing happens. After a few tries I downloaded NoScribe 0.4.0 and it works. But what is the problem for using 0.4.3? My system: MacOS Monterey 12.7.2, MacBook Pro early 2015, 16GB. 
<img width="290" alt="Bildschirmfoto 2024-01-27 um 17 11 26" src="https://github.com/kaixxx/noScribe/assets/157977717/c35b8129-5cb7-44ca-8363-49a8e65c19fe">

**Comment by gernophil** [comment]
Some internal packages have been updated from 0.4.0 to 0.4.3 so I guess it might be an incompatibility with macOS 12.7.2 and one of these packages. Can you open the Terminal (`/System/Applications/Utilities/Terminal.app`) and type `/Applications/noScribe.app/Contents/MacOS/noScribe`. We might get a better error message here.

**Comment by motormox** [comment]
Hey, I have the same problem with 0.4.4 on a MacOS Monterey 12.7.2. I am not super sure how terminals work, but I typed the link you provided in my terminal and it says this: 

![Skærmbillede 2024-02-23 kl  12 52 30](https://github.com/kaixxx/noScribe/assets/161029999/bcf2a058-7c6c-4232-96d1-b5085c278442)

CharlottesAir2:~ moxi$ /Applications/noScribe.app/Contents/MacOS/noScribe
[4470] Error loading Python lib '/Applications/noScribe.app/Contents/Frameworks/Python': dlopen: dlopen(/Applications/noScribe.app/Contents/Frameworks/Python, 0x000A): Symbol not found: (_mkfifoat)
  Referenced from: '/Applications/noScribe.app/Contents/Frameworks/Python.framework/Versions/3.9/Python'
  Expected in: '/usr/lib/libSystem.B.dylib'
CharlottesAir2:~ moxi$ 

- I will try to download the older version (0.4.0) too, I hope it works then, but maybe let me know if you are able to identify the problem.
Thanks for the support!

**Reply by gernophil** [reply]
I'm not absolutely sure, but this seems to be an issue with macOS older then 13 and cpython. I have to dig a little deeper, but it might not work, if I built noScribe with a recent macOS and try to run it with macOC 12. Here are some related links:
https://stackoverflow.com/questions/76223414/pyinstaller-bundle-crashing-on-macos-12-due-to-python

https://github.com/python/cpython/issues/97897
